ansible/roles/ckan/files/override-requirements.txt

```diff
@@ -1,3 +1,3 @@
 flask==0.12 # https://github.com/ckan/datapusher/issues/165
 flask-login==0.3.0 # https://github.com/ckan/datapusher/issues/165
-simplejson==3.12.10 # https://github.com/ckan/ckan/issues/4299#issuecomment-399597263
+simplejson==3.12.0 # https://github.com/ckan/ckan/issues/4299#issuecomment-399597263
```

Comment:
Should we upgrade to the newest version? Apparently 3.16.0 is the current release (https://pypi.org/project/simplejson/)